enterprise/internal/github_apps/store/store.go

```diff
@@ -90,16 +96,30 @@ func (s *gitHubAppsStore) Create(ctx context.Context, app *ghtypes.GitHubApp) (i
 	if err != nil {
 		return -1, err
 	}
+
+	baseURL, err := url.Parse(app.BaseURL)
```

Comment:
It looks like buildkite is complaining this `err` is unused, and I guess it has a point. We should probably bail out early if we couldn't parse the base URL. Maybe:

```suggestion
	baseURL, err := url.Parse(app.BaseURL)
	if err != nil {
		return -1, errors.New("unable to parse base URL: %s", baseURL)
	}
```